contracts/HybridCustody.cdc

```diff
@@ -245,6 +246,10 @@ pub contract HybridCustody {
         pub fun removeChild(addr: Address) {
             if let cap = self.accounts.remove(key: addr) {
                 // TODO: Add access(contract) methods that flow down to ChildAccount s.t. parent is removed if exists in ChildAccount.parents
+                let acct = cap.borrow()
+                    ?? panic("cannot remove invalid account")
```

Comment:
I don't believe we'll want to panic here, otherwise a broken Capability will prevent removal from the Manager.

We could just chain the call, similar to how we get the optional id below.

Reply:
+1 here, we should not do anything which will prevent the parent account being able to remove something. If the capability is invalid, it's on the owner of the child account to clean things up